### PR TITLE
.mailmap, .organizationmap: add Fufei, Mingqiao and Ying Affiliation

### PR DESCRIPTION
**Pull request by heyingstar**
Ying He's PR #16836
wumingqiao's PR #16553
shangfufei's PR #16006

Signed-off-by: Ying He <heyingbj@inspur.com>

### CODE REVIEW

**Comment by heyingstar** [comment]
@joscollin Please help me review this. Thank you.

File: .mailmap [hunk 1]
```diff
@@ -360,6 +360,7 @@ Sébastien Han <shan@redhat.com> <seb@redhat.com>
 Sebastien Ponce <sebastien.ponce@cern.ch> <Sebastien.Ponce@cern.ch>
 Sebastien Ponce <sebastien.ponce@cern.ch> root <root@lxbre43a05.cern.ch>
 Sergey Arkhipov <nineseconds@yandex.ru>
+shangfufei <shangfufei@inspur.com>
```
Comment (theanalyst) [inline .mailmap]: can you use the real name here (I belive Shang Fu Fei ?)

File: .mailmap [hunk 2]
```diff
@@ -432,6 +433,7 @@ Wenjun Huang <wenjunhuang@tencent.com>
 Wido den Hollander <wido@42on.com> <wido@widodh.nl>
 Wu Xingyi <wuxingyi@letv.com>
 Wei Qiaomiao <wei.qiaomiao@zte.com.cn>
+wumingqiao <wumingqiao@inspur.com>
```
Comment (theanalyst) [inline .mailmap]: here too

File: .mailmap (hunk 1 above)
Comment (joscollin) [inline .mailmap]: Is this a github username `shangfufei`? If so, please update your actual name, which is usually the Display Name in your email.

File: .mailmap (hunk 2 above)
Comment (joscollin) [inline .mailmap]: ditto

File: .organizationmap [hunk 3]
```diff
@@ -257,6 +257,9 @@ Inktank <contact@inktank.com> Tyler Brekke <tyler.brekke@inktank.com>
 Inktank <contact@inktank.com> Warren Usui <warren.usui@inktank.com>
 Inktank <contact@inktank.com> Yehuda Sadeh <yehuda@inktank.com>
 Inktank <contact@inktank.com> Yuri Weinstein <yuri.weinstein@inktank.com>
+Inspur <contact@inspur.com> shangfufei <shangfufei@inspur.com>
+Inspur <contact@inspur.com> wumingqiao <wumingqiao@inspur.com>
```
Comment (joscollin) [inline .organizationmap]: ditto

**Comment by heyingstar** [comment]
@joscollin done! Thanks.

File: .mailmap [hunk 4]
```diff
@@ -269,6 +270,7 @@ Michael Rodriguez <michael@newdream.net>
 Michael Rodriguez <michael@newdream.net> <michael@squid.newdream.net>
 Michal Koutný <mkoutny@suse.com>
 Min Chen <minchen@ubuntukylin.com>
+Mingqiao Wu <wumingqiao@inspur.com>
```
Comment (joscollin) [inline .mailmap]: As per the current version of .mailmap, this is in the incorrect alphabetical order.

Comment (heyingstar) [reply .mailmap]: @joscollin Why? Can you help me change this.Thank you.
First name is 'Mingqiao'.
Last name is 'Wu'.
Thank you very much.

Comment (joscollin) [reply .mailmap]: I meant, This is the correct order:
```
Min Chen
MingXin Liu
Mingqiao Wu
Mingyue Zhao 
```

I don't know how a capital letter came in between for `MingXin`. But that entry was already existing anyway.

**Comment by heyingstar** [comment]
@joscollin I see. Thank you.I changed it right now.
Thank you.

**Pull request merged**